include/UartSerial.h

```diff
@@ -76,6 +79,11 @@ class UartSerial : public Stream {
     struct termios _tio_original_config;
     struct termios _tio_config;
     serialEvent _bytes_available_callback;
+    void *_callbackContext;
+    
+    struct pollfd _fds[1];
```

Comment:
yes, it can be an array of pollfd structs but we will only ever use one. Changing it to no longer be an array.